src/dosext/mfs/mfs.c

```diff
@@ -4212,7 +4212,7 @@ static int dos_fs_redirect(struct vm86_regs *state)
       if (offset > 0)
         offset = -offset;
       offset = lseek(fd, offset, SEEK_END);
-      Debug0((dbg_fd, "Seek returns fs=%d ofs=%lld\n", fd, (long long)offset));
+      Debug0((dbg_fd, "Seek returns fd=%x ofs=%lld\n", fd, (long long)offset));
```

Comment:
What is "incorrect" with that?

Reply:
Did I misunderstand, was 'fs' intentional, and five lines earlier it was printed as hex?

Reply:
Ah, I thought you only replaced d to x, thanks.